tests/validation/tests/v3_api/test_create_ha.py

```diff
@@ -201,6 +224,23 @@ def install_cert_manager():
     time.sleep(120)
 
 
+def install_eks_ingress():
+    run_command_with_stderr(export_cmd + " && kubectl apply -f " +
+                            DATA_SUBDIR + "/nlb2.yml")
+
+
+def set_route53_with_eks_ingress():
+    kubectl_ingress = "kubectl get ingress -n cattle-system -o " \
+                      "jsonpath=\"" \
+                      "{.items[0].status.loadBalancer.ingress[0].hostname}\""
+    ingress_address = run_command_with_stderr(export_cmd
+                                              + " && " +
+                                              kubectl_ingress).decode()
+    AmazonWebServices().upsert_route_53_record_cname(RANCHER_HA_HOSTNAME,
+                                                     ingress_address)
+    time.sleep(20)
```

Comment:
@izaac We should wait for the DNS resolution to succeed instead of a sleep of 20 sec. In the last couple of runs we have done , seems like 20 seconds is not enough.